packages/xo-server-perf-alert/src/index.js

```diff
@@ -440,6 +453,7 @@ ${monitorBodies.join('\n')}`
         relatedNode[l.name] = l
       })
       const parser = typeFunction.createParser(
+        definition.comparator,
```

Comment:
What about existing settings?
How this plugin will behave after update?

Reply:
The plugin will be configured and validated with the schema in its registration, that's why it will take the schema default value.

Reply:
Are you sure?

Have you tried purging the config, configure the old version of the plugin, upgrade and run it without reconfiguring?